src/stories/Library/reservation-header/ReservationHeader.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import ReservationHeader from "./ReservationHeader";
+
+export default {
+  title: "Library / Reservation header",
+  component: ReservationHeader,
+  decorators: [withDesign],
+  argTypes: {},
+  parameters: {},
```

Comment:
Should there be a url for the design? and are argTypes and parameters necessary?

Reply:
On it! :)